R/layer-sf.R

```diff
@@ -45,7 +45,7 @@ LayerSf <- ggproto("LayerSf", Layer,
     # automatically determine the name of the geometry column
     # and add the mapping if it doesn't exist
     if ((isTRUE(self$inherit.aes) && is.null(self$computed_mapping$geometry) &&
-         is.null(plot$computed_mapping$geometry)) ||
+         is.null(self$computed_mapping$geometry)) ||
```

Comment:
nope - read the whole statement again :-)

Reply:
A right, now we're testing this twice. Though now `plot$computed_mapping` cannot reasonably exist, so we can best skip this.